Tasks/MavenAuthenticateV0/mavenauth.ts

```diff
@@ -55,6 +51,58 @@ async function run(): Promise<void> {
             tl.setVariable('FIRST_RUN_SETTINGS_XML_EXISTS_PATH', userSettingsXmlPath);
         }
 
+#if WIF
+        const entraWifServiceConnectionName = tl.getInput("workloadIdentityServiceConnection");
+        const feedIdNames = tl.getDelimitedInput("artifactsFeeds", ',');
+
+        if (entraWifServiceConnectionName) {
+            
+            tl.debug(tl.loc("Info_AddingFederatedFeedAuth", entraWifServiceConnectionName));
+            let token = await getFederatedWorkloadIdentityCredentials(entraWifServiceConnectionName);
+            
+            if (token) {
+
+                const centralElement = {
+                    id: 'central',
+                    username: 'CentralBuildWIF',
+                    password: token
+                }
```

Comment:
Token generation code for internal and service connection lives in mavenutil. Move these there as well

Reply:
I'm not authenticating "central" specifically anymore, users who want this should include it as input

Reply:
Move the federated token generation into utils as well. I dont think moving the system access token out here does not make sense. 

Not sure what central has do here?

Reply:
Central is outdated. From teams conversation I went back to adding auth when going through feeds